src/observable/bindCallback.ts

```diff
@@ -9,6 +9,24 @@ import {AsyncSubject} from '../subject/AsyncSubject';
 export class BoundCallbackObservable<T> extends Observable<T> {
   subject: AsyncSubject<T>;
 
+  /* tslint:disable:max-line-length */
+  static create<TResult>(callbackFunc: (callback: (result: TResult) => any) => any, selector?: void, scheduler?: Scheduler): () => Observable<TResult>;
```

Comment:
I don't know how important that is to be honest.  My feeling is we don't need to, because generally no one is going to use the Observable class, they're almost always going to go through the method.